CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## NEXT
 
+### Fixdd
```

Comment:
Fixit?

Reply:
😄

Reply:
En Percy laat ook wat gekke dinge zien.

Reply:
Zie betoog in #1164, we accepteren de huidige functionaliteit van Percy